src/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: center;
+  width: 200px;
+  box-sizing: border-box;
+}
+
+.card__imac_image {
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product_name {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Do you need this? Not sure I remember the problem from Q&A correctly, but you could try `width: 100%`